reference/import_export/product-export.rst

```diff
@@ -161,7 +148,7 @@ As a product may not have values for all attributes, depending on the product, t
 Product Writer
 --------------
 
-This element receives the products as arrays and writes the lines in a csv file.
+This element receives products as standard format, converts them in flat format with the converter and writes the lines in a csv file.
```

Comment:
probably yes, but I don't think here is the part to explain what is it ? What do you think to create another page to explain what a converter is ?